botorch/acquisition/objective.py

```diff
@@ -90,8 +90,7 @@ class ScalarizedPosteriorTransform(PosteriorTransform):
     scalarize: bool = True
 
     def __init__(self, weights: Tensor, offset: float = 0.0) -> None:
-        r"""Affine posterior transform.
-
+        r"""
```

Comment:
I deleted these since Sphinx is just pasting the `__init__` docstring at the end of the class docstring, creating weirdness like this:
![image](https://user-images.githubusercontent.com/9137425/179614741-aa52d41e-6998-41fe-9cee-8d66f79480f3.png)
and
![image](https://user-images.githubusercontent.com/9137425/179614886-5eda698e-1588-451a-8393-cf615c5a2ae4.png)

Reply:
Interesting, I wasn't aware of that. Looks like sphinx is not a big fan of having docstrings both in the class and in `__init__`. @Balandat thoughts on having sphinx outputs look prettier vs having the docstrings be more informative? My preference is on the richer docstring side, though I am biased since I read the code much more often than I look at the API reference.

Reply:
I could go either way on this. I like the more expressive/informative docstrings, but I agree that what Sphinx spits out is kind of weird.

Reply:
I'm not a big fan of the redundancy of having the same text in two places, since it would be pretty easy to forget to update it in both places.